### PR TITLE
Search movies

### PR DESCRIPTION
**Pull request by ZacharyEllixson**
Creating feature 3 functionality and tests

### CODE REVIEW

**Comment by jikingg** [comment]
👍

**Review by jikingg — APPROVED** [review]

File: app.py
Comment (WKirklandJr) [inline app.py:1]: Looks good, good job implementing the get_movie_by_title

File: tests/e2e/test_search_movies_page.py
Comment (WKirklandJr) [inline tests/e2e/test_search_movies_page.py:1]: Good idea for no query, real query & fake query

**Review by WKirklandJr — APPROVED** [review]
Good job, code looks good

**Pull request merged**